example/customer/customer.go

```diff
@@ -0,0 +1,310 @@
+	err = customerService.UpdateCustomerUser(*updateCustomerUserParams)
+
+	if err != nil {
+		fmt.Printf("error updating customer user: %v\n", err)
+		return
+	}
+
+	// Delete Customer User
+	deleteCustomerUserParams := customer.NewDeleteCustomerUserParams()
+	deleteCustomerUserParams.Customer = *customerResponse
+	deleteCustomerUserParams.CustomerUser = *customerUserResponse
+	err = customerService.DeleteCustomerUser(*deleteCustomerUserParams)
+
+	if err != nil {
+		fmt.Printf("error deleting customer user: %v\n", err)
+		return
+	}
+
+	// Delete Customer
+	deleteParams := customer.NewDeleteCustomerParams()
+	deleteParams.Customer = *customerResponse
+	err = customerService.DeleteCustomer(deleteParams)
+
+	if err != nil {
+		fmt.Printf("error deleting customer: %v\n", err)
+		return
+	}
+
+	fmt.Printf("Customer deleted. Account number: %s", customerResponse.HexID)
+}
+
+func ConfigureCustomerAccount(
+	customerService customer.CustomerService,
+	customerResponse customer.GetCustomer,
+) error {
+
+	// Get customers available services
+	getCustomerServicesParams := customer.NewGetCustomerServicesParams()
+	getCustomerServicesParams.Customer = customerResponse
+	services, err := customerService.GetCustomerServices(
+		*getCustomerServicesParams,
+	)
+
+	if err != nil {
+		fmt.Printf("error retrieving available customer services: %v\n", err)
+		return err
+	}
+
+	// DEBUG delete
+	fmt.Println(services)
+
+	var serviceIDs []int
+	for _, service := range *services {
+		if service.Name == "HTTP Large Object" || service.Name == "HTTPS Large Object" {
+			serviceIDs = append(serviceIDs, service.ID)
+		}
+	}
+
+	fmt.Printf("HTTP large service IDs identified: %v", serviceIDs)
+
+	// Update customer services
+	updateCustomerServicesParams := customer.NewUpdateCustomerServicesParams()
+	updateCustomerServicesParams.Customer = customerResponse
+	updateCustomerServicesParams.ServiceIDs = serviceIDs
+	updateCustomerServicesParams.Status = 1 // 1 Enable, 0 Disable
+	err = customerService.UpdateCustomerServices(*updateCustomerServicesParams)
+
+	if err != nil {
+		fmt.Printf("error updating customer services: %v\n", err)
+		return err
+	}
+
+	fmt.Printf("Enabled services with IDs: %v", serviceIDs)
+
+	// Get customer delivery region
+	getCustomerDeliveryRegionParams := customer.NewGetCustomerDeliveryRegionParams()
+	getCustomerDeliveryRegionParams.Customer = customerResponse
+	deliveryRegionID, err := customerService.GetCustomerDeliveryRegion(
+		*getCustomerDeliveryRegionParams,
+	)
+
+	if err != nil {
+		fmt.Printf("error retriving customer delivery region: %v\n", err)
+		return err
+	}
+
+	fmt.Printf("Currently enabled regionID: %v", deliveryRegionID)
+
+	// Update delivery regions
+	updateCustomerDeliveryRegionParams := customer.NewUpdateCustomerDeliveryRegionParams()
+	updateCustomerDeliveryRegionParams.Customer = customerResponse
+	updateCustomerDeliveryRegionParams.DeliveryRegionID = 2 // North America and Europe
```

Comment:
Would it make sense to enumerate these in the SDK?

Reply:
Yeah, I like this suggestion. I am going to create a ticket to followup on this. We can improve the experience greatly by adding this for more than just Delivery Regions. 

e.g. Services, Access Modules